src/core/Akka.Tests/Util/TaskHelpers.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Akka.Tests.Util
+{
+    public static class TaskHelpers
```

Comment:
Might not be a bad idea to move this into `AkkaSpec` at some point in the future